Traceback (most recent call last):
  File "dys.py", line 72, in <module>
IOError: [Errno 22] invalid mode ('r') or filename: ''
